pkg/controllers/vdb/install_reconciler.go

```diff
@@ -69,6 +70,16 @@ func (d *InstallReconciler) Reconcile(ctx context.Context, req *ctrl.Request) (c
 	if err := d.PFacts.Collect(ctx, d.Vdb); err != nil {
 		return ctrl.Result{}, err
 	}
+	// We generate https conf file and skip the install phase when running
+	// the vclusterOps feature flag
+	if vmeta.UseVClusterOps(d.Vdb.Annotations) {
+		if ok, podNotRunning := d.PFacts.anyPodsNotRunning(); ok {
+			d.Log.Info("At least one pod isn't running.  Aborting the install.", "pod", podNotRunning)
+			return ctrl.Result{Requeue: true}, nil
+		}
+		return ctrl.Result{}, d.generateHTTPCerts(ctx)
```

Comment:
To keep the same level of abstraction in this function, can we move these contents to its own function? It might make sense to create the new function as `installForVClusterOps` and rename `analyzeFacts` to `installForAdmintools`.